### PR TITLE
add confirm dialog for revert function

### PR DESCRIPTION
**Pull request by heyagosper**

### CODE REVIEW

File: src/client/app/site-info/site-info.component.ts [hunk 1]
```diff
@@ -256,6 +257,24 @@ export class SiteInfoComponent implements OnInit, OnDestroy {
...
 
+    /**
+     * Shows a dialog prompting users to confirm reverting/reloading the page
+     */
+	public confirmRevert(): Promise<boolean> {
+	      let msg: string = 'You have made changes to the ${this.siteId} Site Log. \
+	      	Reverting will reload the page and will lose any unsaved changes.';
+	      return new Promise<boolean>((resolve, reject) => {
+	          this.dialogService.confirmRevertDialog(msg,
+	              function () {
```
Comment (lbodor) [inline src/client/app/site-info/site-info.component.ts]: Let's use fat arrows, () =>

**Comment by lbodor** [comment]
No tabs please.

File: src/client/app/site-info/site-info.component.ts [hunk 2]
```diff
@@ -256,6 +257,24 @@ export class SiteInfoComponent implements OnInit, OnDestroy {
         });
     }
 
+    /**
+     * Shows a dialog prompting users to confirm reverting/reloading the page
+     */
+	public confirmRevert(): Promise<boolean> {
```
Comment (lbodor) [inline src/client/app/site-info/site-info.component.ts]: I don't think you need a promise here. I suspect that the result would be the same without it.

**Comment by frankfu03** [comment]
All good. :+1:

**Comment by lbodor** [comment]
:+1:

**Pull request merged**